string name       # 设备名称
int32 status      # 设备状态